### DISCUSSION TITLE
v18.19.0 & he wants to continue > There is an update available! Updated to Node.js v18.

### DISCUSSION THREAD

**Discussion by sk2eu** [wg-easy/wg-easy#681] [General]
HI 

The system (proxmox>docker>portainer) is up to date and it keeps complaining that there is an update... how can I fix this ?

```
root@docker5-119-OPtext:~# node -v
npm -v
v18.19.0
10.2.5
root@docker5-119-OPtext:~
```

![grafik](https://github.com/wg-easy/wg-easy/assets/48036204/bca12072-1387-4c84-ae36-049223c2147e)

**Comment by mucfun** [comment]
Same Problem here.
Web GUI shows message that "there is an update available" Updated to Node.js v18.
Banner shows on 4 different WireGuard servers (Ubuntu 22.04 on vmware and Ubuntu 22.04 on proxmox)

**Comment by eandersons** [comment]
Did any of you care to click on the button "Update" to see that it points to upgrade instructions?

After pulling newer image (note that `ghcr.io/wg-easy/wg-easy` has to be used instead of `wg-easy/wg-easy`) and recreating container the update notification is not visible anymore.

**Comment by mucfun** [comment]
Thanks a lot ... "ghcr.io/wg-easy/wg-easy" on docker restart did it
Your wireguard tool is absolute fantastic :-)

**Reply by eandersons** [reply]
You are welcome!
WireGuard Easy is not my project, I am just an user. 😃

**Comment by sk2eu** [comment]
yes man the wrong image ... so all is well. thanks colleague
  
```
    image: ghcr.io/wg-easy/wg-easy
    container_name: wg-easy
```
